google-site-kit.php

```diff
@@ -8,15 +8,17 @@
  * @link      https://sitekit.withgoogle.com
  *
  * @wordpress-plugin
- * Plugin Name: Site Kit by Google
- * Plugin URI:  https://sitekit.withgoogle.com
- * Description: Site Kit is a one-stop solution for WordPress users to use everything Google has to offer to make them successful on the web.
- * Version:     1.73.0
- * Author:      Google
- * Author URI:  https://opensource.google.com
- * License:     Apache License 2.0
- * License URI: https://www.apache.org/licenses/LICENSE-2.0
- * Text Domain: google-site-kit
+ * Plugin Name:    Site Kit by Google
+ * Plugin URI:     https://sitekit.withgoogle.com
+ * Description:    Site Kit is a one-stop solution for WordPress users to use everything Google has to offer to make them successful on the web.
+ * Version:        1.73.0
+ * Requires up to: 4.7
```

Comment:
After closer inspection, this was defined incorrectly in the ACs (my mistake!).

This should be `Requires at least`, just like in `readme.txt`. I confused it with "Tested up to".